### PR TITLE
[IT-1234] Allow VPN access

### PR DESCRIPTION
**Pull request by zaro0508**
* Use update EC2 template which includes SG ingress to allow
AWS VPN access to instance.
* Update parameters to match new template

depends on https://github.com/Sage-Bionetworks/aws-infra/pull/338

### CODE REVIEW

File: config/prod/belltown-solly-2019.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Provision an EC2 instance
 template:
   type: "http"
-  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.2.7/managed-ec2-linux-v1.j2"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.9/EC2/jc-ec2-linux.j2"
 stack_name: "belltown-solly-2019"
```
Comment (ahayden) [inline config/prod/belltown-solly-2019.yaml:5]: I believe this host got terminated

**Review by ahayden — APPROVED** [review]
This will also pick up other changes to the ec2 template in aws-infra

File: config/prod/belltown-solly-2019.yaml (hunk 1 above)
Comment (zaro0508) [reply config/prod/belltown-solly-2019.yaml]: you are correct, it's removed.

**Review by xschildw — APPROVED** [review]
Bring in new So they just pick up the extra CIDR?

**Pull request merged**